docs/protocol/accounts/permissions.md

```diff
@@ -0,0 +1,46 @@
+# Permissions
```

Comment:
Should this section be called "Authorization" perhaps?

Reply:
changed all permission wording to authorization(s)

Reply:
Thank you